tests/object_sync_test.go

```diff
@@ -710,6 +887,168 @@ func makeTestUsers(t *testing.T) []User {
 	return users
 }
 
+type Notification struct {
+	Name        string            `                                  icingadb:"name"`
+	HostName    *string           `icinga2:"host_name"               icingadb:"host.name"`
+	ServiceName *string           `icinga2:"service_name"            icingadb:"service.name"`
+	Command     string            `icinga2:"command"                 icingadb:"notificationcommand.name"`
+	Times       map[string]string `icinga2:"times"`
+	Interval    int               `icinga2:"interval"                icingadb:"notification_interval"`
+	Period      *string           `icinga2:"period"                  icingadb:"timeperiod.name"`
+	//Zone      string                   `icinga2:"zone"                    icingadb:"zone.name"`
+	Types      value.NotificationTypes  `icinga2:"types"                   icingadb:"types"`
+	States     value.NotificationStates `icinga2:"states"                  icingadb:"states"`
+	Users      []string                 `icinga2:"users"`
+	UserGroups []string                 `icinga2:"user_groups"`
+	Vars       *CustomVarTestData       `icinga2:"vars"`
+
+	utils.VariantInfo
+}
+
+func (n Notification) fullName() string {
+	if n.ServiceName == nil {
+		return *n.HostName + "!" + n.Name
+	} else {
+		return *n.HostName + "!" + *n.ServiceName + "!" + n.Name
+	}
+}
+
+func (n Notification) verify(t require.TestingT, db *sqlx.DB) {
+	verifyIcingaDbRow(t, db, n)
+
+	{
```

Comment:
I'd add a comment before each of the following three block that states the table it checks.

Reply:
Done